doc/source/contributing/write_examples.rst

```diff
@@ -0,0 +1,42 @@
+How to write a good example?
+============================
+
+Here are some tips for writing a good example.
+
+Use ``mapdl.convert_script``
+----------------------------
+
+If you want to translate a MAPDL script into a PyMAPDL one, you can first
+run the following commands:: 
```

Comment:
```suggestion
run these commands:: 
```
Get rid of "the following" if a code block immediately follows and a colon introduces it. You won't believe how many unnecessary "followings" occur!